recipes/mitoz/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "3.0a0" %}
```

Comment:
We try to avoid alpha releases, since they're not intended for use except by the developers.

Reply:
Thanks! I follow your suggestion and further update the software.

Now it falls back to Python 3.6 and updates to 3.0.

I have created a new pull request.